poc/ts/src/DebugBecomingArguments.ts

```
@@ -0,0 +1,103 @@
+   */
+  private getParameterNames(func: Function): string[] {
+    const fnStr = func.toString();
+    const result = fnStr.slice(fnStr.indexOf('(')+1, fnStr.indexOf(')')).match(/([^\s,]+)/g);
+    return result || [];
+  }
+
+  /**
+   * Logs details about a parameter
+   */
+  private logParameterDetails(
+    constructor: Function,
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)